testsuite/features/support/constants.rb

```diff
@@ -196,166 +196,326 @@
+                                      'openSUSE Leap 15.5 (aarch64)' => 'openSUSE-Leap-15.5-aarch64-uyuni'
+                                    } }.freeze
 
 # Used for creating bootstrap repositories
-# The values can be found under Admin -> Setup Wizard -> Products
+# The values can be found under Admin -> Setup Wizard -> Products for SUMA
 # Select the desired product and have a look at its product channels
 # The required product has to be synced before.
-PARENT_CHANNEL_TO_SYNC_BY_BASE_CHANNEL = { 'SLE-Product-SUSE-Manager-Proxy-4.3-Pool' => 'sle-product-suse-manager-proxy-4.3-pool-x86_64',
-                                           'SLES12-SP4-Pool' => nil,
-                                           'SLES12-SP5-Pool' => nil,
-                                           'SLES15-SP1-Pool' => 'sle-product-sles15-sp1-pool-x86_64',
-                                           'SLES15-SP2-Pool' => 'sle-product-sles15-sp2-pool-x86_64',
-                                           'SLES15-SP3-Pool' => 'sle-product-sles15-sp3-pool-x86_64',
-                                           'SLES15-SP4-Pool' => 'sle-product-sles15-sp4-pool-x86_64',
-                                           'SLES15-SP5-Pool' => 'sle-product-sles15-sp5-pool-x86_64',
-                                           'SUSE-MicroOS-5.1-Pool for x86_64' => 'suse-microos-5.1-pool-x86_64',
-                                           'SUSE-MicroOS-5.2-Pool for x86_64' => 'suse-microos-5.2-pool-x86_64',
-                                           'SLE-Micro-5.3-Pool for x86_64' => 'sle-micro-5.3-pool-x86_64',
-                                           'SLE-Micro-5.4-Pool for x86_64' => 'sle-micro-5.4-pool-x86_64',
-                                           'almalinux9 for x86_64' => nil,
-                                           'Fake Base Channel' => nil,
-                                           'RHEL x86_64 Server 7' => 'rhel-x86_64-server-7',
-                                           'EL9-Pool for x86_64' => 'el9-pool-x86_64',
-                                           'oraclelinux9 for x86_64' => nil,
-                                           'RHEL8-Pool for x86_64' => nil,
-                                           'rockylinux-9 for x86_64' => nil,
-                                           'ubuntu-2004-amd64-main' => nil,
-                                           'ubuntu-2204-amd64-main' => nil,
-                                           'debian-10-pool' => 'debian-10-pool-amd64',
-                                           'debian-11-pool' => 'debian-11-pool-amd64',
-                                           'openSUSE-Leap-15.4-Pool for aarch64' => nil,
-                                           'openSUSE-Leap-15.5-Pool for aarch64' => nil }.freeze
+# For Uyuni, you have to use `spacewalk-common-channels -l` with the appended architecture
+# e.g. almalinux9 -> almalinux9-x86_64
+PARENT_CHANNEL_TO_SYNC_BY_BASE_CHANNEL = { 'SUSE Manager' =>
```

Comment:
This is not exactly the parent channel, but the parent channel label... I think we better differentiate between channel name and channel label, for instance here:

`'SLE-Micro-5.4-Pool for x86_64' => 'sle-micro-5.4-pool-x86_64'`

The key is a channel name, while the value is a channel label.

Reply:
This would be something to improve in another PR.